doc/source/data/inspecting-data.rst

```diff
@@ -143,3 +144,53 @@ of the returned batch, set ``batch_format``.
 For more information on working with batches, see
 :ref:`Transforming batches <transforming_batches>` and
 :ref:`Iterating over batches <iterating-over-batches>`.
+
+
+Inspecting runtime statistics
+=============================
+
+Ray Data calculates statistics during execution like runtime and memory usage for the different stages. 
+
+View debug stats for your Dataset executions via :meth:`Dataset.stats() <ray.data.Dataset.stats>` on an executed dataset. The stats are also persisted under `/tmp/ray/session_*/logs/ray-data.log`.
```

Comment:
```suggestion
To view debug stats about your `:class:<~ray.data.dataset.Dataset>` executions, call :meth:`Dataset.stats() <ray.data.Dataset.stats>` on an executed dataset. The stats are also persisted under `/tmp/ray/session_*/logs/ray-data.log`.
```

<img width="203" alt="image" src="https://github.com/ray-project/ray/assets/26107013/9a4e27b6-f295-4e2f-9128-2bd6579cc7bd">

Style guide says don't use via.

This is the first reference to `Dataset` in this section, so let's cross-reference it?